packages/cli/test/dev/fixtures/08-hugo/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "private": true,
+  "engines": {
+    "node": "16.x"
```

Comment:
For these `vc dev` fixtures specifically, the node version needs to jive with the version of Node the GH Actions runner is using, since we launch `vc dev` directly in the container.